website/static/js/accountSettings.js

```diff
@@ -344,6 +348,48 @@ var ExternalIdentityViewModel = oop.defclass({
     }
 });
 
+var ChangeDefaultStorageLocation = oop.defclass({
+    constructor: function() {
+        this.client = new UserProfileClient();
+        this.profile = ko.observable(new UserProfile());
+        this.locationSelected = ko.observable({'name': ''});
+
+        this.locationSelectedName = ko.computed(function () {
+            return this.locationSelected().name;
+        }, this);
+
+
+        this.client.fetch().done(
+            function(profile) {
+                this.profile(profile);
+                this.locationSelected(this.profile().default_storage_location());
+            }.bind(this)
+        );
+
+
+    },
+    urls: {
+        'change': '/api/v1/profile/region/'
```

Comment:
Prefer the verb "update" to "change" for PUT requests (see the other `urls` properties in this file).

Reply:
Along the same lines, `UpdateDefaultStorageLocation` may be a better name for this class.